tests/test_performance.py

```diff
@@ -0,0 +1,16 @@
+from lona.html import Tr, Td
+
+
+class TestPerformance:
```

Comment:
You wrote a unit-test style test. Could you port it to a pytest-style test (just a function, without the class)

Reply:
Fixed in ec0315a43357b753d9ac2a57b0200bba0d6aee68

Reply:
Done in f71d682c99cd6b9048e8d49b4d90e8bfbb091669